process/OpenHW-Specification-Process.md

```diff
@@ -0,0 +1,152 @@
+# OpenHW Process Document: Specification Process, States and Format
+
+## Revision History
+
+This document desbribes OpenHW's specification process and format.
+It was adopted by consensus in the TWG meeting of July 2023.
+
+The V1.0 of this document reflects the agreements made at that meeting and adds additional details. 
+
+| Date        | Version | Notes                                                 |
+| :---------- | :------ | :---------------------------------------------------- |
+| 22 Jan 2024 | 1.0     | Formal draft reflecting July 2023 google docs			|
+                                  
+## Specification Process
+
+### Overview
+
+OpenHW uses a lightweight specification development process to produce specification documents that are developed and frozen in a designated Task Group, then ratified and released by TWG.
+
+### Requirements for OpenHW Specification Process
+
+
+#### Basics
+
+| Number 	| Issue 									| Comment                                               							|
+| :---		| :------ 									| :-------------------------------------------------------------------------------- |
+| 1 		| Nature of the specification				| A collective derivative work from the individual member contributions, together with staff contributions	|																			
+| 2 		| Editors of the specification				| OpenHW via its members and staff													|
+| 3 		| Contributions by non-members				| For further study				|
+| 4 		| Eclipse Specification Process				| Not used				|
+
+#### Contributions into the Specification
+
+| Number 	| Issue 									| Comment                                               							|
+| :---		| :------ 									| :-------------------------------------------------------------------------------- |
+| 1    		| License options for contributions: 		|																					|
+| 1.1		| Apache 2.0								| As per OpenHW Member Agreement 													|
+| 1.2		| Solderpad 2.1								| As per OpenHW Member Agreement 													|
+| 1.3		| Commons Attribution Share Alike 4.0 International license | Agreed by OpenHW CEO April 2023									| 
+| 2			| Patent Grant included in contribution license:  | 																				|
+| 2.1		| Apache 2.0								| Yes 																				|
+| 2.2		| Solderpad 2.1								| Yes																				|
+| 2.3		| Commons Attribution Share Alike 4.0 International license | No																| 
```

Comment:
Requirement 2.3 contradicts 1.3.  Are there precedent rules that resolve this contraction?

Reply:
@MikeOpenHWGroup I'm not sure I understand the point of this change

Reply:
Just to make it clear that Requirement 2.3 does not contradict 1.3. Currently, the wording is identical so the distinction is not obvious.